classes/Domain/Services/TalkEmailer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Domain\Services;
+
+use Swift_Mailer;
+use Swift_Message;
+
+class TalkEmailer
+{
+    /**
+     * @var \Swift_Mailer
+     */
+    private $swiftMailer;
+
+    /**
+     * @param Swift_Mailer $swiftMailer
+     */
+    public function __construct(Swift_Mailer $swiftMailer)
+    {
+        $this->swiftMailer = $swiftMailer;
+    }
+
+    /**
+     * @param Swift_Message $message
+     *
+     * @return int
+     */
+    public function send(Swift_Message $message)
+    {
+        try {
+            return $this->swiftMailer->send($message);
+        } catch (\Exception $e) {
+            echo $e->getMessage();
+            die();
```

Comment:
Are we sure we just want to die here, rather than 

* letting the exception bubble up
* catching the exception and throwing a different exception (one that hides the underlying implementation)

?

Reply:
This seems to be "best practise" when sending emails…

https://github.com/opencfp/opencfp/blob/fab87b36c7c5a950eb53d33d6dc042309edc30e0/classes/Domain/Services/ResetEmailer.php#L63-L70

I was already wondering about this when I came about that other piece of code. We should probably log the exception and display an error message to the user instead. Since there are more `die`s in the codebase, should we file a cleanup task for that?

Reply:
@derrabus 👍